kowl/values.yaml

```diff
@@ -133,6 +133,9 @@ secret:
     kafka-tls-passphrase: "kafka-tls-passphrase"
     kafka-sasl-password: "kafka-sasl-password"
 
+  extraData: []
```

Comment:
Could extra data not also be a map?

Reply:
Or basically any valid yaml?

Reply:
I think it is a map already.You would use it like this:
```
extraData:
  foo: bar
  bar: baz
```

Maybe it is not a good idea to set an empty list as default

Reply:
If you allow any valid yaml, then how do you get the key value?. Can you provide an example?

Reply:
Ah ok, then I would make a map as default.

Reply:
You mean the default value? Because it is a map already. A list would look like this right?
```
extraData:
 - foo: bar
   bar: baz
 - foobar: foobaz
```

Reply:
Yes but your default value for `extraData` is `[]` but you expect it to be a `{}` so I would change it to `{}`